src/RadioButtonGroup/index.jsx

```diff
@@ -1,37 +1,54 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
Since `RadioButton` should `bind` it's event handler, it can no longer be a functional component (right?). However, I don't think it should live in another file, as these two components are (right now at least) always going to be coupled.